classes/filter_manager.php

```diff
@@ -129,7 +116,7 @@ public static function get_static_filters() {
      *
      * @return array Of filter data from the DB, though properties is already json_decoded.
      */
-    public function get_user_filters() {
+    public function get_course_filters() {
```

Comment:
Again, please do not change the public methods. Some developers may have built things on top of the plugin and if they relied on those their code would break without notice. Unless there is a very good reason, I'd suggest to keep the methods. And if their names need to change, the older method needs to remain and be an alias to the new one, or its behaviour needs to be maintained.